antora-playbook.yml

```diff
@@ -3,7 +3,10 @@ site:
   # features that depend on the site URL: https://docs.antora.org/antora/2.3/playbook/site-url/#when-should-the-site-url-be-set
   url: https://documentation.bonitasoft.com/
   start_page: bonita::index.adoc
-  robots: allow
+  robots: |
+    User-agent: *
+    Allow: /0/archives/release-notes*
+    Disallow: */0/archives*
```

Comment:
❓ I guess they are disallowed as we have configured redirects from old bonita 5.x versions to the archive pages. And we won't to drop references to the old versions in the Search Engine.
It is the reason of the change?

Reply:
Hello, the reason is detail in this  Primelis document : https://docs.google.com/presentation/d/1xr9hc-sYbVZ3PTrjP_OjV9JDEo-NGuEx5eXgVY_NRxI/edit
Pages 21 to 26 and 29

We will must change urls path for archives pages.
The change in the robots.txt is the first step.
Actual change on the file has no effect on the presence in search engine.